src/libmongoc/src/mongoc/mongoc-bulkwrite.h

```diff
@@ -255,6 +255,8 @@ mongoc_bulkwrite_set_client(mongoc_bulkwrite_t *self, mongoc_client_t *client);
 // `*session` may be modified when `mongoc_bulkwrite_execute` is called.
 MONGOC_EXPORT(void)
 mongoc_bulkwrite_set_session(mongoc_bulkwrite_t *self, mongoc_client_session_t *session);
+MONGOC_EXPORT(bool)
+mongoc_bulkwrite_is_acknowledged(mongoc_bulkwrite_t const *self);
```

Comment:
Also add a .rst file to document the new public function. Though CONTRIBUTING.md could use an update, it includes commands for [building docs](https://github.com/mongodb/mongo-c-driver/blob/d77a79e4b83bbcc4bc48a02cf443e4879fb36161/CONTRIBUTING.md#documentation):

```
uv run --frozen --with 'sphinx-autobuild' sphinx-autobuild -b html src/libmongoc/doc/ src/libmongoc/doc/html --re-ignore ".*.pickle" --re-ignore ".*.doctree" -j auto
```